components/Changers/FontSizeChanger.jsx

```diff
@@ -0,0 +1,25 @@
+import styles from "@/styles/components/changers/font-size-changer.module.scss";
+
+export default function FontSizeChanger({ fontSize, setFontSize }) {
```

Comment:
The state should be in here (descibed about it more when talked about the context/custom hook)

Reply:
changed?